File: 2.0/java/me/getAccount.java
import com.callfire.api.client.CallfireClient;
import com.callfire.api.client.api.account.model.Account;

class ApiClientSample {
    public static void main(String[] args) {
        CallfireClient callfireClient = new CallfireClient("api_login", "api_password");
        Account account = callfireClient.meApi().getAccount();
    }
}

File: 2.0/csharp/me/getAccount.cs
using CallfireApiClient;
using CallfireApiClient.Api.Account.Model;

public class ApiClientSample
{
    public static void Main(string[] args)
    {
        var client = new CallfireClient("api_login", "api_password");
        UserAccount account = client.MeApi.GetAccount();
    }
}
